evap/static/ts/src/datagrid.ts

```diff
@@ -416,12 +429,23 @@ export class ResultGrid extends DataGrid {
             });
         }
 
+        for (const [name, { slider }] of this.filterSliders.entries()) {
+            this.state.rangedFilter.set(name, slider.range);
+
+            slider.onRangeChange = () => {
```

Comment:
Afaik, for text inputs in the search field we have debouncing/throttling. When I drag a slider, does it trigger a range change for intermediate values, or will a range change event only be triggered for the final value when dropping the slider?

Reply:
the slider (and the range) is updated constantly. I thought about "debouncing" the slider maybe if the performance is too bad

Reply:
As discussed: No strong opinion on what/how much we debounce, but I think it should be consistent with all user-provided filters that can have accidental "intemediate" values.

Reply:
we now debounce the slider update events which trigger only the row updates